packages/app/obojobo-repository/shared/api-util.js

```diff
@@ -0,0 +1,38 @@
+const APIUtil = {
+	get(endpoint, format) {
```

Comment:
I'd like to not have this duplicated here and in obojobo-document-engine.  Let's find a way to pull out the reusable parts of APIUtil (get, post, delete, etc?) into a shared library.